app/views/user_mentee_applications/mentee_application_states/_phone_screen_scheduled.html.erb

```diff
@@ -0,0 +1,5 @@
+<h3>Phone Screen</h3>
+
+<p>We would like to schedule a zoom call with you. This call is a quick interview with one of our senior members. It will last about 30 minutes. We will have read your application and reviewed your code. The ideal call is an informal conversation about your goals and your coding journey. There will also be an opportunity for you to ask us questions as well.</p>
```

Comment:
Maybe this open with something about how we reviewed their code challenge and liked it?